src/main/java/com/devathon/preguntonic/dto/RoomPlayerInitInfo.java

```diff
@@ -0,0 +1,14 @@
+/*
+ *
+ * Copyright (c) 2024 Devathon., All Rights Reserved.
+ *
+ */
+package com.devathon.preguntonic.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public record RoomPlayerInitInfo(
```

Comment:
This class does not convince me at all, since where we use it we are assigning it 2 responsibilities, I think we can manage it, only creating the room in the endpoint where it is used and then when doing the join, always creating the user, reusing the same logic of the `/room.join/`